deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -36,53 +36,49 @@
         <h4 class="panel-title pull-left">Current Deploy</h4>
     </div>
     {% if env|isEnvEnabled %}
-    {% if env.envName != "ngapp2-A" and env.envName != "ngapp2-B" %}
-	<div class="row">
-        <a href="/env/{{ env.envName }}/{{ env.stageName }}/new_deploy/"
-            type="button" class="deployToolTip btn btn-default btn-block"
-            data-toggle="tooltip" title="Create a new deploy in current environment">
-            <span class="glyphicon glyphicon-file"></span> Create Deploy
-        </a>
-    </div>
-    {% endif %}
-    {% if has_deploy %}
-    {% if env.envName != "ngapp2-A" and env.envName != "ngapp2-B" %}
-	<div class="row">
-        <a href="/env/{{ env.envName }}/{{ env.stageName }}/rollback/"
-            type="button" class="deployToolTip btn btn-default btn-block"
-            title="Rollback to a previous deploys">
-            <span class="glyphicon glyphicon-repeat icon-flipped"></span> Rollback
-        </a>
-    </div>
-    {% endif %}
-    {% if stages|length > 1 and env.envName != "ngapp2-A" and env.envName != "ngapp2-B" %}
-	<div class="row">
-        <a href="/env/{{ env.envName }}/{{ env.stageName }}/promote/{{ env.deployId }}"
-            type="button" class="deployToolTip btn btn-default btn-block"
-            title="Promote current deploy to different stages">
-            <span class="glyphicon glyphicon-transfer"></span> Promote
-        </a>
-    </div>
-    {% endif %}
-    {% if pinterest and env.envName != "ngapp2-A" and env.envName != "ngapp2-B" %}
-    <div class="row">
-        <a href="/env/{{ env.envName }}/{{ env.stageName }}/hotfixes/"
-            type="button" class="deployToolTip btn btn-default btn-block"
-            data-toggle="tooltip" title="Create a hotfix based on current deploy">
-            <span class="glyphicon glyphicon-fire"></span> Hotfix
-        </a>
-    </div>
+        <div class="row">
+            <a href="/env/{{ env.envName }}/{{ env.stageName }}/new_deploy/"
+                type="button" class="deployToolTip btn btn-default btn-block"
+                data-toggle="tooltip" title="Create a new deploy in current environment">
+                <span class="glyphicon glyphicon-file"></span> Create Deploy
+            </a>
+        </div>
 
-    <div id="privateBldUploadBtnId" class="row">
```

Comment:
so this will disable the private build upload feature. on the UI we may also point users to the wiki or tell them to get help from #teletraan if they need enable private build (possible future work).

Reply:
actually, the diff in github is not the same as phab --> The reason it shows removed is because i indented the code & below you can see the code still exists.

Reply:
Here it is in action:

1. my deploy 
![image](https://user-images.githubusercontent.com/29519/76488056-92ce3380-63e1-11ea-9d20-dfb79bb5a0ce.png)

2. ![image](https://user-images.githubusercontent.com/29519/76488032-834eea80-63e1-11ea-8310-83dd2d5f01f4.png)